packages/web/src/common/store/upload/sagas.ts

```diff
@@ -405,9 +405,9 @@ export function* handleUploads({
         artworkSource: track.metadata.artwork?.source,
         genre: track.metadata.genre,
         moode: track.metadata.mood,
-        size: track.file.size,
-        type: track.file.type,
-        name: track.file.name,
+        size: track.file?.size,
+        type: track.file?.type,
+        name: track.file?.name,
```

Comment:
curious why these have to change to optional now? something to do with the edit track is on its own page instead of a modal?